packages/layer-composer/src/generators/types.ts

```diff
@@ -65,7 +65,7 @@ export interface BasemapGeneratorConfig extends GeneratorConfig {
   /**
    * Sets the color of the map background in any format supported by Mapbox GL, see https://docs.mapbox.com/mapbox-gl-js/style-spec/types/#color
    */
-  basemap?: string
+  basemap: BasemapType
```

Comment:
I think we could leave it as optional because we already have a default configuration